pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -264,17 +259,44 @@ func NewCloud(configReader io.Reader) (cloudprovider.Interface, error) {
 			config.CloudProviderRateLimitBucketWrite)
 	}
 
+	// Conditionally configure resource request backoff
+	if config.CloudProviderBackoff {
+		// Assign backoff defaults if no configuration was passed in
+		if config.CloudProviderBackoffRetries == 0 {
+			config.CloudProviderBackoffRetries = backoffRetriesDefault
+		}
+		if config.CloudProviderBackoffDuration == 0 {
+			config.CloudProviderBackoffDuration = backoffDurationDefault
+		}
+		// Log warning for deprecated options.
+		if config.CloudProviderBackoffExponent != 0 {
+			klog.Warning("Azure cloud provider config 'cloudProviderBackoffExponent' has been deprecated. 2 is always used as the backoff exponent.")
```

Comment:
I don't think we can do this, we should print the deprecation warning, but use the user supplied value...

Reply:
I do think that's the right way, but we couldn't do it because the exponent is fixed `2` within SDK.